nncf/quantization/layers.py

```diff
@@ -230,17 +240,40 @@ def __init__(self, config):
         self.collect_scale_statistics = False
         if self.per_channel:
             self.scale_shape = get_per_channel_scale_shape(self.input_shape, self.is_weights)
-        self.scale = nn.Parameter(torch.ones(self.scale_shape), requires_grad=True)
-        self.eps = 1e-16
+        self.scale_tensor = nn.Parameter(torch.ones(self.scale_shape), requires_grad=True)
```

Comment:
Mark the internal parameter with a leading underscore. `_scale_tensor`

Reply:
ok